monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java

```diff
@@ -647,7 +647,12 @@ public ExecutionIdentifier eval(final RepositoryFile file) {
                         ExecutionIdentifier result = file.readFile(new Func<InputStream, ExecutionIdentifier>() {
                             @Override
                             public ExecutionIdentifier eval(InputStream in) {
-                                return SaxExecutionIdentifierReader.read(in, file.getQualifiedPath());
+                                try {
+                                    return SaxExecutionIdentifierReader.read(in, file.getQualifiedPath());
+                                } catch (Exception e) {
+                                    logger.error("The file {} could not be read" + e, file.getQualifiedPath());
```

Comment:
The errors level seems excessive. error is usually reserved for fatal problem, this is more of a problem that doesn't really affect the program.